### PR TITLE
add idr-py develop to dependencies

### PR DESCRIPTION
**Pull request by bramalingam**
Removing outdated idr-py from the path and adding pip installation step for idr-py repo.

@joshmoore @manics

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,10 +33,8 @@ RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
         gseapy \
         py2cytoscape \
         pydot \
-        tqdm
-
-# Add idr-notebook library to path
-RUN echo /notebooks/library > /opt/conda/envs/python2/lib/python2.7/site-packages/idr-notebooks.pth
+        tqdm \
+        git+https://github.com/IDR/idr-py@develop
```
Comment (manics) [inline Dockerfile:37]: Travis failed, this needs to be `@master`

**Review by manics — CHANGES_REQUESTED** [review]

**Comment by manics** [comment]
This is all included in #13

File: Dockerfile (hunk 1 above)
Comment (bramalingam) [reply Dockerfile:37]: I assumed it would be develop, cause am opening the PR against develop

Comment (bramalingam) [reply Dockerfile:37]: ahh wrong repo branch, yeah will change it to master

**Comment by bramalingam** [comment]
This is already included in https://github.com/IDR/jupyter-docker/pull/13, so closing this PR in favour of the other one.

**Pull request closed without merging**